src/adminPage/adminPage.component.tsx

```diff
@@ -0,0 +1,232 @@
+import React, { useState } from 'react';
+import Typography from '@material-ui/core/Typography';
+import {
+  Theme,
+  StyleRules,
+  createStyles,
+  WithStyles,
+  withStyles,
+  Paper,
+  TextareaAutosize,
+  FormControlLabel,
+  Checkbox,
+  Button,
+} from '@material-ui/core';
+import { connect } from 'react-redux';
+import { StateType } from '../state/state.types';
+import {
+  AppStrings,
+  MaintenanceState,
+  ScheduledMaintenanceState,
+} from '../state/scigateway.types';
+import { getString, getAppStrings } from '../state/strings';
+import { ThunkDispatch } from 'redux-thunk';
+import { AnyAction } from 'redux';
+import {
+  setMaintenanceState,
+  setScheduledMaintenanceState,
+} from '../state/actions/scigateway.actions';
+
+const styles = (theme: Theme): StyleRules =>
+  createStyles({
+    root: {
+      padding: theme.spacing(2),
+    },
+    titleText: {
+      color: theme.palette.secondary.main,
+      fontWeight: 'bold',
+    },
+    paper: {
+      display: 'flex',
+      flexDirection: 'column',
+      marginTop: theme.spacing(2),
+      marginBottom: theme.spacing(2),
+      padding: theme.spacing(2),
+      [theme.breakpoints.up(800 + theme.spacing(4))]: {
```

Comment:
I don't think this is working for me. Below 832px (window width) the width scales with the window as I'd expect, and above 864px the width of the component is fixed to 800px. But for windows between 832px and 864px the right edge of the component is cropped off. I think it would just be a case of increasing `theme.spacing(4)` to `theme.spacing(8)`, presumably it needs to account for the padding of the root element and the paper element?

![Screen Shot 2021-03-01 at 17 04 42](https://user-images.githubusercontent.com/61705287/109531947-758deb80-7ab0-11eb-86af-8c88afdd1beb.png)

Reply:
Thanks, that works!